src/views/direction/direction.dot

```diff
@@ -27,3 +27,5 @@
   </div>
   {{= this.roadMapPanel.render() }}
 </div>
+
+<button class="direction_shortcut" {{= click(this.open, this) }}></div>
```

Comment:
Maybe this button should be part of `ExtendedControl`, just like the geolocation button ?

The position is currently fixed, and is broken after the itinerary preview has been enabled:
![image](https://user-images.githubusercontent.com/4726554/53821984-1eabb500-3f6f-11e9-81c0-ba83d297b6c3.png)

Reply:
The bug shown in your screenshot is fixed: when opening the itinerary panel (or a route preview), the direction button is hidden.

but I'll need help for moving the button in ExtendedControl (why is there no .dot template for them?)